content/articles/Factor-that-affects-the-performance-deep-learning-systems/index.md

```diff
@@ -0,0 +1,114 @@
+-	Programming Languages like Python, Java, C++
+-	Calculus
+-	Probability
+-	Statistics
+-	Linear Algebra
+-	[TensorFlow](https://www.google.com/url?sa=t&source=web&rct=j&url=https://www.tensorflow.org/&ved=2ahUKEwi15PyaroL1AhWmS2wGHZZUBdMQFnoECF4QAQ&sqi=2&usg=AOvVaw0TGZBeXHx2CVPI2FiDZclR) 
+-	[Microsoft Cognitive Toolkit](https://docs.microsoft.com/en-us/cognitive-toolkit/) 
+-	[Pytorch](https://www.google.com/url?sa=t&source=web&rct=j&url=https://pytorch.org/&ved=2ahUKEwi4lMHSr4L1AhXozIUKHVkeAO4QFnoECAUQAQ&usg=AOvVaw2mABY6VbqZdRJYnleMzDSb) 
+-	[Keras](https://www.google.com/url?sa=t&source=web&rct=j&url=https://keras.io/about/&ved=2ahUKEwjX16yCrYL1AhUGyoUKHaoMBqMQFnoECCoQBQ&usg=AOvVaw3frsnniocU9GHfkfWqfuMJ) 
+
+### The Concept of Deep Learning
+In deep learning, an excessive volume of information [data sets](  https://www.simplilearn.com/what-is-data-article) can be trained at the same time. This uses learning algorithms in deep learning. Deep learning systems tend to gather functional facts from larger datasets. 
+
+Artificial neural networks can classify data based on responses to a series of binary true or false questions involving highly complicated mathematical calculations while processing the data. 
+
+A facial recognition algorithm, for example, learns to detect and recognize face edges and lines, then more significant aspects of the faces, and finally overall representations of faces. The algorithm learns and improves with time, increasing the likelihood of getting it right. In this situation, the facial recognition software will recognize faces properly over time.
+
+The ability to fine-tune the budget and the considered problem are two enhancing variables in Deep Learning. Any optimizer’s accuracy is measured using these factors.
+
+### Some Determinants That Transform Deep Learning Systems’ Performance
+
+
+ #### 1.  Taking into account the issue (considered problem) that the system will be tasked with resolving; 
+ 
+-	What kind of system should be built? 
+-	How will the developed algorithm handle the immediate user’s demands?
+-	 How easily will re-engineering be made possible?
+
+#### 2.   Evaluation difficulties of default hyperparameters for multiple optimisers.
+ Poor [accuracy](https://sigopt.com/blog/common-problems-in-hyperparameter-optimization/) is obtained here as a result of low awareness from training the system data sets. The [Adams optimiser](https://www.google.com/url?sa=t&source=web&rct=j&url=https://machinelearningmastery.com/adam-optimization-algorithm-for-deep-learning/&ved=2ahUKEwjok7uPlIL1AhWBOewKHbYnDcMQFnoECDUQAQ&usg=AOvVaw3DyIH6zRqOJ31iqVVAy1z-)  is used in this deep system algorithm, which is adaptable. 
+
+#### 3.  Technical expertise in deep learning system network design.
+The demand and need for deep learning experts is highly recommended here. Experience is needed to increase the learning rate and improve the system's performance.
```

Comment:
You can't really state this point, since you're talking about the systems, not about the people that build systems. Although what you said was right, it doesn't fit this context